include/db/postgres.inc.php

```diff
@@ -89,7 +89,12 @@ function serendipity_db_reconnect() {
  * @return  string   output string
  */
 function serendipity_db_escape_string($string) {
-    return pg_escape_string($string);
+    global $serendipity;
+
+    if (is_null($string))
+        return $string;
+
+    return pg_escape_string($serendipity['dbConn'], $string);
```

Comment:
Wasn't this another issue/patch? Unsure how it affects existing setups...?

Reply:
Yes, there sadly seem to be some commits that got mixed together :/